Src/GoogleApis.Auth.DotNet4/OAuth2/ServiceAccountCredential.cs

```diff
@@ -66,12 +70,27 @@ public Initializer(string id)
                 : this(id, GoogleAuthConsts.TokenUrl) { }
 
             /// <summary>Constructs a new initializer using the given id and the token server URL.</summary>
-            public Initializer(string id, string tokenServerUrl) : base(tokenServerUrl)
+            public Initializer(string id, string tokenServerUrl)
+                : base(tokenServerUrl)
             {
                 Id = id;
                 Scopes = new List<string>();
             }
 
+            /// <summary>Constructs a new initializer using the given id and the private key in the ClientCredentialParameters.</summary>
+            public Initializer(ClientCredentialParameters clientCredentialParameters)

```

Comment:
Getting the ADC feature checked in and available to our consumers is also an immediate goal.